hooks/autoLogin.ts

```
@@ -9,6 +9,7 @@ import { deleteToken, getRefreshToken, setAccessToken, setRefreshToken } from '@
 import { useMemberStore } from '@/zustand/member/member';
 import { useHasLifeStyleStore } from '@/zustand/member-stat/member-stat';
 import { useHasRoomStore } from '@/zustand/room/room';
+import { useAuthProvider } from '@/providers/AuthProvider';
```

ESLint

[error] `@/providers/AuthProvider` import should occur before import of `@/utils/token`

(import/order)